.editorconfig

```diff
@@ -0,0 +1,7 @@
+[*.rs]
+end_of_line = lf
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
It means I don't need to change my vim config for different repos (slash copy an editorconfig in from somewhere when freshly cloning the repo), it picks this file up and automatically adjusts tab width etc. Also means if anyone else uses an editor that understands .editorconfig they'll get the same as well.